.github/workflows/mulighetsrommet-api.yaml

```diff
@@ -32,22 +32,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Set up JDK 11
-        uses: actions/setup-java@v3
-        with:
-          distribution: temurin
-          java-version: 11
-      - name: Setup Gradle
-        uses: gradle/gradle-build-action@v2
-      - name: Set up test environment
-        run: docker-compose up -d
+      - name: Setup backend
+        uses: ./.github/actions/setup-backend
       - name: Build with Gradle
         run: ./gradlew build
-      - name: Publish test report
-        uses: ScaCap/action-surefire-report@v1
```

Comment:
Nå som vi fjerner denne, har vi noe alternativ? Siden den ikke fungerer som den skal uansett så er jeg enig at den skal bort. Jeg bare tenker på nå om man må kjøre testene lokalt for å finne ut av hva som feiler. Virker tungvint.